test/evergreen.yml

```diff
@@ -2130,6 +2130,17 @@ tasks:
           #run for 2 hours ( 2 * 60 = 120 minutes), use default config
           format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 120
 
+  - name: format-stress-test-pull-request
+    tags: ["pull_request"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+      - func: "format test script"
+        vars:
+          # run for 10 minutes, number of parallel jobs equal to number of cores in machine,
```

Comment:
A few things:
- Will you add `smp_command: -j $(grep -c ^processor /proc/cpuinfo)` to `vars` rather than hardcoding the value in the argument list? 
- I'd prefer to remove the comment. I don't think it adds any value.

Reply:
@tammybailey : Yes, that's a good idea, fixed the hardcoded number and removed redundant comments.